Packs/ContentTesting/Scripts/UnitTestCasePrep/UnitTestCasePrep.py

```diff
@@ -1,13 +1,14 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
+# Final Test: 6.10
 import io
 
 
 def main():
-    args = demisto.args()
-    testName = args.get("testName", "")
-    listName = args.get("listName", "")
     try:
+        args = demisto.args()
+        testName = args.get("testName", "")
```

Comment:
Why did you remove these 3 lines in to the 'try' block?
It does not need to be there beucus there is no scenario for it to fail, and the way it is now will not execute,  because in the bottom the 'except Exception' is out of the 'try' block  and we call 'testName', which is inside of the 'try' block.

Reply:
Same here. No lines removed and exception still valid?

Reply:
Sorry, please ignore this.

Reply:
Hi @rurhrlaub, how are you?
I've messaged you via Slack. Is "David Uhrlaub" your username there?
I'm currently looking into this unusual problem of lines moving without your touch, and would greatly value your responses.